src/components/studies/scheduler/SessionStartTab.tsx

```diff
@@ -71,61 +155,103 @@ const SessionStartTab: FunctionComponent<SessionStartTabProps> = ({
+    return editableTextboxErrorState.has(identifier + '-' + index)
+      ? 'red'
+      : 'black'
+  }
+  const calendarIcon = (
+    <img src={CalendarIcon} className={classes.calendarIcon}></img>
+  )
   return (
     <Box className={classes.root}>
       <Box width="600px" mx="auto" textAlign="left">
-        <MTBHeadingH1>Session Start Dates</MTBHeadingH1>
-        <p>
-          Sessions in a study can be scheduled to start after a participant logs
-          into the study for the first time known as “Initial Log in” or by a
-          calendar date known as “Event” that are unique to each participant.
-          Once your study launches, you can specify these dates for each
-          participant in the Participant Manager tab.
+        <p className={classes.paragraphText}>
+          Sessions in a study can be scheduled to start{' '}
+          <strong>after a participant logs into the study</strong> for the first
+          time known as “Initial Log in” or by a <strong>calendar date</strong>{' '}
+          known as “Event” that are unique to each participant. Once your study
+          launches, you can specify these dates for each participant in the{' '}
+          <strong>Participant Manager tab.</strong>
         </p>
-
         <MTBHeadingH2>How will your session(s) start? </MTBHeadingH2>
-        <p>
-          You can add additional calendar Events and relabel <br />
-          them to make it easier to reference later.
+        <p className={classes.paragraphText}>
+          You can add additional calendar Events and{' '}
+          <strong>
+            relabel <br />
+            them to make it easier to reference later.
+          </strong>
         </p>
-        <Box display="flex">
-          <Box flexShrink={0} flexBasis={400} pl={8}>
+        <Box display="flex" justifyContent="flex-start" mt={3}>
+          <Box flexShrink={0} minWidth="200px" mr={2}>
             <>
-              Initial_Login
-              {(study.clientData.events || []).map((evt, index) => (
-                <FormGroup
-                  row={true}
-                  key={evt.identifier + index}
-                  style={{alignItems: 'center', marginTop: '21px'}}>
-                  <EditableTextbox
-                    initValue={evt.identifier}
-                    onTriggerUpdate={(newLabel: string) =>
-                      updateEvent(evt, newLabel)
-                    }></EditableTextbox>
-
-                  <IconButton
-                    edge="end"
-                    size="small"
-                    onClick={() => deleteEvent(evt.identifier)}>
-                    <DeleteIcon></DeleteIcon>
-                  </IconButton>
-                </FormGroup>
+              <Box className={classes.intialLoginContainer}>Initial_Login</Box>
+              {localEventIdentifiers.map((evt, index) => (
+                <Box display="block">
+                  <FormGroup
+                    row={true}
+                    key={index}
```

Comment:
index is not the best key to use especially when you can delete things from the list may be evt. identifier?